modules/openy_features/openy_location/modules/openy_loc_filter/src/Form/LocationFilterSettingsForm.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Drupal\openy_loc_filter\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\node\NodeInterface;
+use Drupal\Core\Database\Connection;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use \Drupal\Core\Cache\Cache;
+
+/**
+* Location Filter settings form.
+*/
+class LocationFilterSettingsForm extends ConfigFormBase {
+
+  const CONFIG_NAME = 'openy_loc_filter.location_filter_settings';
+
+  /**
+   * The database connection.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected $connection;
+
+  /**
+   * Constructs a new LocationFilterSettingsForm.
+   *
+   * @param \Drupal\Core\Database\Connection $database
+   *   Database connection.
+   */
+  public function __construct(Connection $database) {
+    $this->connection = $database;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static($container->get('database'));
+  }
+
+  /**
+  * {@inheritdoc}
+  */
+  public function getFormId() {
+    return 'openy_loc_filter_location_filter_settings';
+  }
+
+  /**
+  * {@inheritdoc}
+  */
+  protected function getEditableConfigNames() {
+    return [
+      self::CONFIG_NAME,
+    ];
+  }
+
+  /**
+  * {@inheritdoc}
+  */
+  public function buildForm(array $form, FormStateInterface $form_state) {
```

Comment:
I think this module should have some description on the Settings page. Let's add an info section on the page. 

**Description:** 
_On this page, you can limit the list of branches and camps, which displayed in popups and filters on the programs and schedules page. This form will affect only the filters which are related to the scheduling, classes, and sessions. By default, all branches and camps displayed. If you want to limit the list, you should choose only those you want to show._